packages/utils-helpers/src/getVersion.js

```diff
@@ -0,0 +1,5 @@
+const getVersion = () => {
+    return 'v1.0.3';
```

Comment:
maybe also add commit hash, so if it's on master, we could identify the correct commit?